### ISSUE TITLE
QGIS 3.12 Merge Vector Layers Box behind main screen. 

### ISSUE THREAD

**Issue by GideonRichards** [qgis/QGIS#38141]
**Describe the bug**

**How to Reproduce**

**QGIS and OS versions**

**Additional context**

**Comment by GideonRichards** [comment]
> **Describe the bug**
> 
> **How to Reproduce**
> 
> **QGIS and OS versions**
> 
> **Additional context

**Comment by GideonRichards** [comment]
New technology and not good with GitHub -  Aargh!!

This item is still open and apologises for my incompetence.

MacOS Catalina and QGIS 3.12 When using the merge layer tool Vector>Data Management Tools > Merge Vector Layers the first box ask for section of vectors and directory.  When these are selected and the OK button is clicked, it looks like nothing is happening and it disappears and it didn't work.  What I have found out is that the next stage dialogue box goes behind the main map window and is not visible.  I have cursed for a long time before finding this.  I think and am no programmer that the code needs to be changed to bring the box to the front.  Hope this can be fixed. Gideon
